autopush/websocket.py

```diff
@@ -365,15 +368,31 @@ def _check_other_nodes(self, result):
             attrs = result.get("Attributes")
             node_id = self._get_aval(attrs.get("node_id", {}), 'S')
             uaid = self._get_aval(attrs.get("uaid", {}), 'S')
-            connected_at = self._get_aval(attrs.get("connected_at", {}), 'N')
-            url = "%s/notif/%s/%s" % (node_id, uaid, connected_at)
-            if not getattr(self, 'testing', False):
-                d = self.ap_settings.agent.request(
-                    "DELETE",
-                    url.encode("utf8"),
-                ).addCallback(self.finish_hello)
-                d.addErrback(self.log_err, extra="Failed to delete old node")
-                return
+            last_connect = self._get_aval(attrs.get("connected_at", {}), 'N')
```

Comment:
So, thing to remember here, last_connect != this_connect, because last_connect is the prior connected_at time, not this one. This makes it somewhat hard to reason about the logic below for me.